app/views/browse/_common_details.html.erb

```diff
@@ -38,4 +38,8 @@
   <% end %>
 </div>
 
-<%= render :partial => "browse/tag_details", :object => common_details.tags %>
+<%= if defined?(@old_features) && @old_features
```

Comment:
This shouldn't be necessary in Ruby. Instance variables (starting with `@`) resolve to `nil` if not defined.


```suggestion
<%= if @old_features
```

Reply:
Suggestion taken in 8b1fb0bf6.